app/forms/csv_upload_form.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class CSVUploadForm
```

Comment:
```suggestion
class CsvUploadForm
```

Follow our naming convention here: https://nimblehq.co/compass/development/code-conventions/#convention-19
The `CSV` should be `Csv`

Reply:
Thank you. I like this Ruby style naming more.